guides/graphite-cli/cli-command-reference.md

```diff
@@ -224,7 +238,7 @@ All `gt repo` commands can also be accessed by the shortcut `gt r`.
     * Show a rough estimate of progress through deleting branches. (This is commonly used by users running `gt repo sync` in a long-standing git repo with tens/hundreds of dead branches where Graphite has just been initialized.)
   * `-f`\
     `--force`
-    * Don't prompt confirmation when Graphite suggests to delete a branch already-merged into main.
+    * Skip prompting to delete feature branches that have been merged into trunk. (may include deleting already merged branches and setting branch parents).
```

Comment:
Let's add the note about the default action instead. Otherwise LGTM

Reply:
These are the default actions? Delete merged branches and rebase. This is also the language used in `--help` of `repo sync`